_includes/v22.2/ui/insights.md

```diff
@@ -0,0 +1,224 @@
+- **Rows Processed**: The total number of rows read and written.
+- **Retries**: The number of times the statement execution was [retried]({{ link_prefix }}transactions.html#automatic-retries).
+- **Contention Time**: The amount of time the statement execution spent waiting in [contention]({{ link_prefix }}performance-best-practices-overview.html#transaction-contention).
+- **Full Scan**: Whether the execution performed a full scan of the table.
+- **Transaction Execution ID**: The ID of the transaction execution for the statement execution.
+- **Transaction Fingerprint ID**: The ID of the transaction fingerprint for the statement execution.
+
+#### Statement execution details
+
+The statement execution details view provides more details on a statement execution insight.
+
+{% if page.cloud != true -%}
+The following screenshot shows the execution details of the statement execution in the preceding section:
+
+<img src="{{ 'images/v22.2/statement_execution_details.png' | relative_url }}" alt="Statement execution details" style="border:1px solid #eee;max-width:100%" />
+{% endif -%}
+
+The **Insights** column shows the name of the insight, in this case **Suboptimal Plan**; the **Details** column provides details on the insight; and the final column contains a **Create Index** button. Click the **Create Index** button to perform a query to mitigate the cause of the insight, in this case to create an index on the ride `start_time` that stores the `rider_id`.
+
+- **Start Time**: The timestamp when the statement execution started.
+- **End Time**: The timestamp when the statement execution ended.
+- **Elapsed Time**: The time that elapsed during statement execution.
+- **Rows Read**: The total number of rows read.
+- **Rows Written**: The total number of rows written.
+- **Transaction Priority**: The [priority]({{ link_prefix }}transactions.html#set-transaction-priority) of the transaction for the statement execution.
+- **Full Scan**: Whether the execution performed a full scan of the table.
+- **Session ID**: The ID of the [session]({{ link_prefix }}ui-sessions-page.html) the statement was executed from.
+- **Transaction Fingerprint ID**: The ID of the transaction fingerprint for the statement execution.
+- **Transaction Execution ID**: The ID of the transaction execution for the statement execution.
+- **Statement Fingerprint ID**: The fingerprint ID of the statement fingerprint for the statement execution.
+
+## Schema Insights tab
+
+{% if page.cloud != true -%}
+To display this view, click **Insights** in the left-hand navigation of the DB Console and select  **Schema Insights**.
+{% endif -%}
+{% if page.cloud == true -%}
+To display this view, click **Insights** in the left-hand navigation of the Cloud Console  and select **Schema Insights**.
+{% endif -%}
+
+This view lists the indexes that have not been used and should be dropped, and/or the ones that should be created, altered, or replaced (based on statement execution).
```

Comment:
"indexes" here could be a link to our reference docs on indexes (I think I did see a link to that from somewhere else on here)